src/model_signing/__init__.py

```diff
@@ -14,4 +14,12 @@
 
 """For the stable high-level API, see model_signing.api."""
 
+from model_signing import hash
+from model_signing import sign
+from model_signing import verify
+
+
 __version__ = "0.2.0"
+
+
+__all__ = ["hash", "sign", "verify"]
```

Comment:
note: this affects what is generated for the documentation site.

Is our plan to only offer documentation for the stable API?

Reply:
Oh, I haven't checked that. But I think only the public part should be documented, I'll mark the rest as private in a future PR.